GeneratorInterface/Configuration/python/GeneratorInterface_EventContent_cff.py

```diff
@@ -52,3 +52,9 @@
         'keep *_genParticles_*_*'
     )
 )
+
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(GeneratorInterfaceRAW,
+                     outputCommands = GeneratorInterfaceRAW.outputCommands+['keep edmHepMCProduct_LHCTransport_*_*'])
```

Comment:
If I've understood correctly, the `LHCTransport` module is enabled to be run here
https://github.com/cms-sw/cmssw/blob/cb972603936bfec65fbc92d3dc66bb4480f0db20/SimPPS/Configuration/python/GenPPS_cff.py#L18-L19

It would be better to use the more specific modifiers. I see the `run3_common` being used also here
https://github.com/cms-sw/cmssw/blob/cb972603936bfec65fbc92d3dc66bb4480f0db20/SimG4Core/Application/python/g4SimHits_cfi.py#L636

If that would be changed to use `ctpps_2021` as well, and the definition of `Run3_pp_on_PbPb`
https://github.com/cms-sw/cmssw/blob/cb972603936bfec65fbc92d3dc66bb4480f0db20/Configuration/Eras/python/Era_Run3_pp_on_PbPb_cff.py#L9
to exclude `ctpps_2021` (assuming that is the intention), then this fragment
https://github.com/cms-sw/cmssw/blob/cb972603936bfec65fbc92d3dc66bb4480f0db20/SimG4Core/Application/python/g4SimHits_cfi.py#L638-L642
would not be needed, nor any explicit treatment for Phase2 (since `ctpps_2021` is excluded from there
https://github.com/cms-sw/cmssw/blob/cb972603936bfec65fbc92d3dc66bb4480f0db20/Configuration/Eras/python/Era_Phase2_cff.py#L21
).